setup.sh

```diff
@@ -1,15 +1,3 @@
-# Use '-i' when sourcing this file to attempt to reset $PATH to system defaults (i.e., based on /etc/profile),
```

Comment:
We haven't used this and I don't think we need it?

Reply:
I guess I can go either way. I'm okay with removing it for now, but if we run into issues again like we had with the ESMF recipe where it was using the calling environment (incl. PATH) rather than the Spack build environment in places, then we should revisit and try to make it work on all systems (I don't remember how far I got with testing, I think it worked on more than just Acorn and Orion but those were the ones that needed added customization).

Reply:
Ok, thanks. I'll take it out for now. I never had to do anything particular about the environment except for Cray systems where you need to have a separate set of modules loaded, depending on the compiler you build for (see Narwhal instructions). Since we've agreed today to move forward with separate environments for different compilers, that will be even less of an issue (again, essentially doing what we already do for Narwhal).